apps/antalmanac/src/components/buttons/Download.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useRef } from 'react';
+import { createEvents } from 'ics';
+import { useSnackbar } from 'notistack';
+import { Button, type ButtonProps, Link, Tooltip } from '@mui/material';
+import { Download as DownloadIcon } from '@mui/icons-material';
+import { getEventsFromCourses, vTimeZoneSection } from '$lib/download';
+import analyticsEnum, { logAnalytics } from '$lib/analytics';
+
+export function DownloadButton(props: ButtonProps = {}) {
+    const ref = useRef<HTMLAnchorElement>(null);
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const exportCalendar = useCallback(() => {
+        const events = getEventsFromCourses();
+
+        createEvents(events, (err, val) => {
+            logAnalytics({
+                category: 'Calendar Pane',
+                action: analyticsEnum.calendar.actions.DOWNLOAD,
+            });
+
+            if (err || !ref.current) {
+                enqueueSnackbar('Something went wrong! Unable to download schedule.', { variant: 'error' });
+                console.log(err);
+                return;
+            }
+
+            // Add timezone information to start and end times for events
+            const icsString = val
+                .replaceAll('DTSTART', 'DTSTART;TZID=America/Los_Angeles')
+                .replaceAll('DTEND', 'DTEND;TZID=America/Los_Angeles');
+
+            // inject the VTIMEZONE section into the .ics file
+            const blob = new Blob([icsString.replace('BEGIN:VEVENT', vTimeZoneSection)], {
+                type: 'text/plain;charset=utf-8',
+            });
+
+            ref.current.href = URL.createObjectURL(blob);
+            ref.current.download = 'schedule.ics';
+            ref.current.click();
+
+            enqueueSnackbar('Schedule downloaded!', { variant: 'success' });
+        });
+    }, [enqueueSnackbar]);
+
+    return (
+        <>
+            <Tooltip title="Download Schedule (.ics)">
```

Comment:
Text and Icon color is blue here, as well as for `Screenshot.tsx` and isn't readable on Dark Mode.

<img width="753" alt="Screenshot 2023-09-01 at 2 33 35 PM" src="https://github.com/icssc/AntAlmanac/assets/100006999/e7f2542d-b952-4de8-8c46-a0f38cf247e2">

Reply:
Oh, yeah the component design is correct; it's supposed to be a minimal abstraction over the original `Button` component. But I just forgot to apply `color="inherit"` when I use them.